### PR TITLE
[P4 tests] [Team 9] Add searchTfIdf Tests 

### PR DESCRIPTION
**Pull request by DanniUCI**
What's your team number?
Team 9

What is the functionality being tested?
getNumDocuments(int segmentNum);
getDocumentFrequency(int segmentNum, String token);
searchTfIdf(List<String> keywords, Integer topK)

Describe your tests briefly:
Test1
Test the functions. Check the correctness of the number of results and the order of results.

Test2
Test the functions. Check the correctness of the number of results and the order of results.
In this test, we have some corner cases. Two documents are very similar, but the score is different.

Does each test case have comments/documentation?
Yes

### CODE REVIEW

File: src/test/java/edu/uci/ics/cs221/index/ranking/Team9TfidfTest.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    private Compressor compressor;
+    private InvertedIndexManager invertedIndex;
+    private static String textUrl1 = "https://raw.githubusercontent.com/DanniUCI/Master-Courses/master/Team9TfidfTest.txt";
+    private static String textUrl2 = "https://raw.githubusercontent.com/DanniUCI/Master-Courses/master/Team9TfidfTest2.txt";
+    static List<String> allDocuments;
+
+    @Before
+    public void setUp() {
+        analyzer = new ComposableAnalyzer(new PunctuationTokenizer(),new PorterStemmer());
+        compressor = new NaiveCompressor();
```
Comment (NinCors) [inline src/test/java/edu/uci/ics/cs221/index/ranking/Team9TfidfTest.java]: Well designed test cases. Maybe switching from naiveCompressor to DeltaCompressor?

**Comment by TLizzie** [comment]
Review from Team 8
Your test case looks good.
